underfs/s3/src/test/java/alluxio/underfs/s3/S3UtilsTest.java

```diff
@@ -44,29 +44,36 @@ public void before() throws Exception {
   }
 
   @Test
-  public void translateOwnerAclTest() {
-    // Grant only READ, READ_ACP permission to the owner. Check the translated mode is 0500.
+  public void translateUserAcl() {
+    // Grant only READ, READ_ACP permission to the user. Check the translated mode is 0500.
     mAcl.grantPermission(mOwnerGrantee, Permission.PERMISSION_READ);
```

Comment:
I think we should be using "owner" instead of "user" everywhere where permissions are concerned